packages/block-library/src/embed/edit.js

```diff
@@ -21,98 +21,42 @@ import classnames from 'classnames';
  * WordPress dependencies
  */
 import { __, sprintf } from '@wordpress/i18n';
-import { Component } from '@wordpress/element';
+import { useState, useEffect } from '@wordpress/element';
```

Comment:
Nitpick: I prefer imports to be in alphabetical order. (There's no standard one way or the other; just a personal preference of mine.)

Example:

```js
import { useEffect, useState } from '@wordpress/element';
import { __, sprintf } from '@wordpress/i18n';
```

Reply:
Either let's make a linting rule, or not bother with this at all. :) I personally don't feel like wasting time on this.